docs/numerical_exp_report.tex

```diff
@@ -137,291 +135,472 @@ \subsection{Tick}
-       Tails Assay & 0.003 & \% $^{235}U$ \\
-       Fuel Enrichment & 4 & \% $^{235}U$ \\
-       Enrichment SWU Capacity & 2000 & $\frac{SWU}{month}$ \\
-       Source Throughput & 3500 & $\frac{kg}{month}$\\
-       Sink Capacity & 200 &  $kg$ \\
-       Number of time steps before facility is decommissioned \footnote{This refers to the number of consecutive time steps that the supply exceeds demand by more than 1 facility output before the facility is decommissioned.} & 3 & timesteps \\
-       \hline
-    \end{tabularx}
-    \end{minipage}
-    \caption {Source, Enrichment and Sink Facility Parameters}
-    \label{tab:everythingelse}
+	\centering
+    \caption {Source, Reactor and Sink Parameters}
+	\label{tab:reactor}
+	\begin{tabular}{|l|l|l|}
+\hline
+\textbf{Source Parameters} & \textbf{Value} & \textbf{Units} \\
+\hline
+Throughput & 1 & kg \\
+Output Commodity & fuel & kg\\
+\hline
+\textbf{Reactor Parameters} & \textbf{Value} & \textbf{Units} \\
+\hline
+Cycle Time & 1 & timesteps \\
+Refuel Time & 0 & timesteps \\
+Lifetime & 1 & timesteps \\
+Power Capacity & 1& MWe \\
+Assembly Size & 1 & kg \\
+\# assemblies per core & 1 & \\
+\# assemblies per batch & 1 & \\
+Input Commodity & fuel & kg\\
+Output Commodity & power & MW\\
+\hline
+\textbf{Sink Parameters} & \textbf{Value} & \textbf{Units} \\
+Throughput & 1 & kg \\
+Input Commodity & spent uox & kg\\
+\hline
+	\end{tabular}
 \end{table}
 
-Table \ref{tab:simulationparameters} provides the input and output commodity amounts and enrichment SWUs required based on a reactor fuel demand of 100 kg for one time step. These input and output commodities are based on the simple demand flow of materials shown in figure \ref{diag:dem}. 
+\pagebreak
+
+\section{Numerical Tests for the Non-optimizing prediction method}
+To ensure that the non-optimizing prediction model is working correctly, a variation of input files that uses the non-optimizing method archetype to deploy or decommission facilities is tested to determine if their outputs match the analytical solution. In this section, the tests that must be met is described based on the parameters defined in table \ref{tab:testscenario} and \ref{tab:reactor} and analytical solution of a defined simple scenario. Unit test examples are included in Appendix B.
+
+The tests are split into test A types and test B types. Test A refers to the test scenarios where facilities are expected to be deployed. Test B refers to the test scenarios where facilities are expected to be decommissioned. 
+
+The prediction algorithm for the non-optimizing method has three user-defined input functionalities. The aim with the various test scenarios are to check if the non-optimizing method archetype will deploy or decommission facilities correctly when there is a variation in the combination of the three input functionalities.  
+\begin{enumerate}
+	\item There is an initial demand. 
+	\item There is an initial facility already present. 
```

Comment:
Maybe we should talk with the group to see if this is something we want (as in don't we want this changed..?)

Also, half of our tests don't assume that there is an initial facility already present, so this may not be valid.

Reply:
I'm not sure what you mean?

Reply:
So it sounds like having the initial facility is a must for the institution, and I'm not sure we should say that it is..?

Also, setting the initial facility is an input parameter of the `institution` in general, not specifically this institution (ie. `NullInst` and `DeployInst` can have initial facility as well)

Reply:
Oh i see what you mean. Would this be more clear? 

```
The prediction algorithm for the non-optimizing method has three user-defined input parameters. The aim with the various test scenarios are to check if the non-optimizing method archetype will deploy or decommission facilities correctly when there is a variation in the combination of the three input parameters. 
1. Initial demand value 
2. Number of Initial facilities
3. Growth rate value 
```

So this implies that initial facility is not a must, but can be defined for the institution.

Reply:
Yeah that sounds good! Maybe also add in 2:
```
2. Number of Initial facilities (supply)
```
Since what the institution really measures is the supply, not the number of facilities.